src/BloomBrowserUI/bookEdit/toolbox/readers/readerToolsModel.ts

```diff
@@ -815,6 +815,12 @@ export class ReaderToolsModel {
     }
 
     var pageStrings = _.values(this.pageIDToText);
+    // In obtaining pageStrings, for some reason lines can be separated by \\n instead of \r\n (or \n?).
```

Comment:
I didn't know where the wierd data was being introduced.  The source of a problem is obviously a better place to fix it.